scripts/buildTests.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs')
+const path = require('path')
+const browserify = require('browserify')
+const fileMapTransform = require('./browserifyFileMapTransform')
+
+const outputFile = process.argv[2]
+
+function listSourceFiles (dir) {
+    return fs.readdirSync(dir).filter(f => f.endsWith('.js')).map(f => path.join(dir, f))
+}
+const input = listSourceFiles(path.join('unit-test', 'background'))
+    .concat(listSourceFiles(path.join('unit-test', 'background', 'classes')))
+    .concat(listSourceFiles(path.join('unit-test', 'background', 'events')))
+    .concat(listSourceFiles(path.join('unit-test', 'background', 'reference-tests')))
+    .concat(listSourceFiles(path.join('unit-test', 'background', 'storage')))
+const b = browserify(input, {
```

Comment:
Nit: Think it would be easier to read if you added a newline and skipped the `b` variable.
```suggestion

browserify(input, {
...
}).bundle(err, bundle) => {
...
})
```
```